src/doc/DocAnnotator.js

```diff
@@ -594,12 +596,8 @@ class DocAnnotator extends Annotator {
             this.modeControllers[TYPES.highlight_comment].applyActionToThreads((thread) => thread.reset(), page);
         }
 
-        let mouseEvent = event;
         // $FlowFixMe
-        if (this.hasTouch && event.targetTouches) {
-            mouseEvent = event.targetTouches[0];
-        }
-        this.lastHighlightEvent = mouseEvent;
+        this.lastHighlightEvent = this.hasTouch && event.targetTouches ? event.targetTouches[0] : event;
```

Comment:
This seems like a good candidate for lodash get. Could help avoid the flow problems as well.